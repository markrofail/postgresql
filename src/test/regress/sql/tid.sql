-- tests for functions related to TID handling

CREATE TABLE tid_tab (a int);

-- min() and max() for TIDs
INSERT INTO tid_tab VALUES (1), (2);
SELECT min(ctid) FROM tid_tab;
SELECT max(ctid) FROM tid_tab;
TRUNCATE tid_tab;

DROP TABLE tid_tab CASCADE;
