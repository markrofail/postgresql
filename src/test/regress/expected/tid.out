-- tests for functions related to TID handling
CREATE TABLE tid_tab (a int);
-- min() and max() for TIDs
INSERT INTO tid_tab VALUES (1), (2);
SELECT min(ctid) FROM tid_tab;
  min  
-------
 (0,1)
(1 row)

SELECT max(ctid) FROM tid_tab;
  max  
-------
 (0,2)
(1 row)

TRUNCATE tid_tab;
DROP TABLE tid_tab CASCADE;
